utils/term_analysis.cpp

```diff
@@ -0,0 +1,58 @@
+/*********************                                                        */
+/*! \file term_analysis.cpp
+** \verbatim
+** Top contributors (to current version):
+**   Makai Mann
+** This file is part of the cosa2 project.
+** Copyright (c) 2019 by the authors listed in the file AUTHORS
+** in the top-level source directory) and their institutional affiliations.
+** All rights reserved.  See the file LICENSE in the top-level source
+** directory for licensing information.\endverbatim
+**
+** \brief Useful functions for term analysis.
+**
+**
+**/
+
+#include "smt-switch/smt.h"
+
+using namespace smt;
+
+namespace cosa
+{
+
+void get_free_symbols(Term & term, UnorderedTermSet & out_symbols)
+{
+  smt::TermVec to_visit({term});
```

Comment:
no need for smt::

Reply:
Done with fcd3fd8a27e3ee5718e33db5d3d3341a94edb2c6